test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_mnist.py

```diff
@@ -191,6 +191,7 @@ def test_smdebug(sagemaker_session, ecr_image, instance_type, framework_version)
 @pytest.mark.model("mnist")
 @pytest.mark.skip_cpu
 @pytest.mark.skip_py2_containers
+@pytest.mark.skip(reason="Consider reworking these tests after re:Invent releases are done")
```

Comment:
Please remove this.

Reply:
this is outdated test and will be fixed afterwards. https://github.com/aws/deep-learning-containers/pull/1044/files#diff-de6e76f81c56a48b429095acaa583f8bf9f9395ed7efd74b7cf24df89951f6eeL198

Reply:
we want to keep it and skip it for now.